app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
This may be a stupid question, but why are we bouncing the user through illiad? Couldn't we handle this request on the back-end, like we do for submitting requests to Symphony?